can-define-stream-kefir.js

```diff
@@ -1,4 +1,5 @@
+var namespace = require('can-util/namespace');
 var canDefineStream = require('can-define-stream');
 var canStreamKefir = require('can-stream-kefir');
 
-module.exports = canDefineStream(canStreamKefir);
+module.exports = namespace.canDefineStreamKefir = canDefineStream(canStreamKefir);
```

Comment:
the `namespace` is setup as `window.can` ... so this would read `can.canDefineStreamKefir`.

This needs to be updated to `namespace.defineStreamKefir`